modules/openthread/platform/radio.c

```diff
@@ -83,7 +83,12 @@ static const struct device *const radio_dev =
 	DEVICE_DT_GET(DT_CHOSEN(zephyr_ieee802154));
 static struct ieee802154_radio_api *radio_api;
 
+/* Get the default tx output power from Kconfig */
+#if defined(CONFIG_OPENTHREAD_DEFAULT_TX_POWER)
+static int8_t tx_power = CONFIG_OPENTHREAD_DEFAULT_TX_POWER;
```

Comment:
`CONFIG_OPENTHREAD_DEFAULT_TX_POWER` should always be defined (`default 0`), not sure why we need to preprocessor check.

Reply:
Which samples fail to build? I assume this is due to another bug?

Reply:
I lost the history of checks here, so I didn't remember. If there isn't any purpose to not include Kconfig.thread in some samples, you're right, there is another bug. Honestly, I don't know all projects that use Open Thread, so I thought there are some that have their own configuration and don't include that file. I can delete that, wait on the CI result and see what's going on.

Reply:
Agreed that there should not be any need for the `#ifdef` here actually. @ArekBalysNordic can you please remove it and we'll see if something fails?